README.md

```diff
@@ -127,8 +127,11 @@ Lucas Lee - playcover@lucas.icu
 
 These open source libraries were used to create this project.
 
-* [inject](https://github.com/paradiseduo/inject)
+* [Inject](https://github.com/paradiseduo/inject)
```

Comment:
`inject` is styled as `inject` not `Inject`

Reply:
I'll revert